page_test.go

```diff
@@ -77,8 +155,7 @@ func TestPageClient(t *testing.T) {
 					}
 					return
 				}
-				// TODO: remove properties from comparing for a while. Have to compare with interface somehow
```

Comment:
It does seem that `reflect.DeepEqual` handles interfaces. Changing the values for the properties causes it to fail.

A nice testing package that I've been using recently is https://github.com/stretchr/testify. Its inputs are interfaces, so it should be able to compare that if you have any concerns about comparing them using reflection.